### PR TITLE
feat: created useHandleNumberFormat function

### PR DESCRIPTION
**Pull request by thetaungg**

### CODE REVIEW

File: src/utils/normalizeToNumberString/__tests__/normalizeToNumberString.test.ts [hunk 1]
```diff
@@ -15,6 +15,20 @@ describe('normalizeToNumberString', () => {
 
         expect(normalizedNumberString).toBe('22.22');
     });
+    it('should give a correct output integer value without prefix', () => {
```
Comment (shashank1010) [inline src/utils/normalizeToNumberString/__tests__/normalizeToNumberString.test.ts:18]: add tests for prefix and post fix.
add test for negative numbers. "-" sign preceded by a number is will be dropped

**Review by shashank1010 — CHANGES_REQUESTED** [review]

**Review by shashank1010 — APPROVED** [review]
